azext_iot/central/services/_utility.py

```diff
@@ -5,13 +5,38 @@
 # --------------------------------------------------------------------------------------------
 # Nothing in this file should be used outside of service/central
 
+from knack.util import CLIError
+from requests import Response
+
 from azext_iot import constants
 from azext_iot.common import auth
 
 
-def get_headers(token, cmd):
+def get_headers(token, cmd, has_json_payload=False):
     if not token:
         aad_token = auth.get_aad_token(cmd, resource="https://apps.azureiotcentral.com")
         token = "Bearer {}".format(aad_token["accessToken"])
 
+    if has_json_payload:
+        return {
+            "Authorization": token,
+            "User-Agent": constants.USER_AGENT,
+            "Content-Type": "application/json",
+        }
+
     return {"Authorization": token, "User-Agent": constants.USER_AGENT}
+
+
+def try_extract_result(response: Response):
+    if response.status_code in [201, 204]:
```

Comment:
nit, A short comment that quickly explains that we don't need to access the body of a 201 or 204 status code response.